addons/im_livechat/tests/test_chatbot_internals.py

```diff
@@ -261,7 +261,6 @@ def get_forward_op_bus_params():
                     (self.cr.dbname, "discuss.channel", discuss_channel.id),
                     (self.cr.dbname, "discuss.channel", discuss_channel.id),
                     (self.cr.dbname, "res.partner", self.partner_employee.id),
-                    (self.cr.dbname, "res.partner", self.env.user.partner_id.id),
```

Comment:
Note: _sync now sends the channel as part of the member update, and only if `is_pinned` actually changes.